plugin/pkg/scheduler/factory/plugins.go

```diff
@@ -46,20 +60,26 @@ type AlgorithmProviderConfig struct {
 	PriorityFunctionKeys util.StringSet
 }
 
-// Registers a fit predicate with the algorithm registry. Returns the name,
-// with which the predicate was registered.
+// RegisterFitPredicate registers a fit predicate with the algorithm
```

Comment:
Random question: Do we have a standard about whether you re-state the function name in the comment? Personally I don't like this style (I'd prefer "Registers a fit predicate..." as it was before) but I guess we don't have a standard practice.